build/configuration/pom.xml

```diff
@@ -149,7 +149,7 @@
       <version.infinispan.maven-plugins>1.0.8.Final</version.infinispan.maven-plugins>
       <version.insights>2.0.4</version.insights>
       <version.io.agroal>2.3</version.io.agroal>
-      <version.io.lettuce>6.3.2.RELEASE</version.io.lettuce>
+      <version.io.lettuce>6.5.2.RELEASE</version.io.lettuce>
```

Comment:
@tristantarrant to confirm that this is ok, we are updating to a different minor version of lettuce in an Infinispan patch release. Can I get your blessing for merging?